### DISCUSSION TITLE
Linux-compatible game but license from Epic Games ?

### DISCUSSION THREAD

**Discussion by KaKi87** [Heroic-Games-Launcher/HeroicGamesLauncher#2285] [Q&A] [answered]
Hello,

A game runs natively on Linux from Steam and Gog, but the license I own is from Epic Games.

Can the app launch the Linux version of the game using my license, or will it have to virtualize the Windows version anyway ?

Thanks

**Comment by Jakdaw** [comment, marked as answer]
No you can't.
